backend/app/main.py

```diff
@@ -181,12 +158,12 @@ def get_eol_message_appid(
         return value
 
 
-@app.get("/projectgroup")
+@app.get("/developer")
 def get_project_groups():
```

Comment:
Should probably also be renamed?

Reply:
Hrm, mixing the project group / developer naming will be confusing at some point I guess